### PR TITLE
Remake: Sec gon give it to ya: Battering Ram

### PR DESCRIPTION
**Pull request by coiax**
It's like #17557, except you can merge it.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Coiax do you want to split some inhands into their own files

**Comment by KorPhaeron** [comment]
or I could do it but I am not sure how to organize it

**Comment by kevinz000** [comment]
Sounds like a pretty big sec buff... to have people going around ramming stuff down anyways.

**Comment by Core0verload** [comment]
No, you can't merge this.

Also it still needs to start respecting bolted, shocked and welded airlocks.

**Comment by optimumtact** [comment]
I can't merge this

**Comment by coiax** [comment]
Splitting inhands should probably be done in its own PR during a spriting freeze. It's okay, sprite conflicts are quick to resolve once you've done them before for a PR.

File: code/game/objects/items/weapons/twohanded.dm [hunk 1]
```diff
@@ -421,6 +421,78 @@
...
+						playsound(A.loc, 'sound/effects/bang.ogg', 50, 1)
+						user.do_attack_animation(A)
+						sleep(20)
+					if(i == 2)
+						if(!proximity)
+							return
+						playsound(A.loc, 'sound/effects/bang.ogg', 75, 1)
+						playsound(A.loc, 'sound/machines/airlock_alien_prying.ogg', 100, 1)
+						var/datum/effect_system/spark_spread/sparks = new
+						sparks.set_up(3, 1, A.loc) //no idea what the 0 is
```
Comment (optimumtact) [inline code/game/objects/items/weapons/twohanded.dm]: there is no zero

Comment (JJRcop) [inline code/game/objects/items/weapons/twohanded.dm]: do not try to bend the zero, that's impossible. instead, only try to realize the truth...

**Comment by KorPhaeron** [comment]
I don't know why the idea of it being counterable in some way is offensive to people

**Comment by optimumtact** [comment]
bolting and welding should make it take longer, shocking should be added.

RCD's should be nerfed

**Comment by Cheridan** [comment]
how did this get so many upvotes the first time and why is it getting downvoted now

**Comment by KorPhaeron** [comment]
Different people paying attention

**Comment by coiax** [comment]
I don't even care that much about this feature, but I'll take care of it in Robustin's absence.

**Comment by coiax** [comment]
Robustin's back in town, I will close this.

**Pull request closed without merging**